### ISSUE TITLE
NTR: Parotid gland biopsy: adenoma (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9117]
New term request Parotid gland biopsy: adenoma (elthat_100323042248): Salivary gland  UMLS: 0747271

**Comment by pnrobinson** [comment]
@MickeySegal
Pleomorphic adenoma seems to be by far the most common parotid adenoma- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4478272/
Should this be referred to here?

**Comment by MickeySegal** [comment]
It is fine to refer to that, but restricting to that would exclude other parotid adenomas such as https://pubmed.ncbi.nlm.nih.gov/2827601/

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000862
- label: Parotid adenoma
- parents: [Salivary gland neoplasm (HP:0100684)](https://hpo.jax.org/app/browse/term/HP:0100684)
- pmid: PMID:26124628
- def: A benign tumor of the parotid salivary gland.
- comment: Pleomorphic adenoma (PA), also known as benign mixed tumor, is the most common salivary tumor, constituting up to two-thirds of all salivary gland neoplasms.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9117
- synonyms:
